framework/web/CHttpSession.php

```diff
@@ -250,7 +250,17 @@ public function setCookieParams($value)
 		extract($data);
 		extract($value);
 		$this->freeze();
-		if(isset($httponly))
+		if(isset($httponly) && isset($sameSite))
```

Comment:
$sameSite should be lowercase here and below, I'll open a PR to fix it.

Reply:
https://github.com/yiisoft/yii/pull/4313

Reply:
Thank you for catching this. Sorry!